### ISSUE TITLE
Containers Broken

### ISSUE THREAD

**Issue by ghsjeff** [EdenPlus/screeps#1]
In func.maintainStructures

var allStructuresInRoom = curRoom.find(FIND_MY_STRUCTURES);

FIND_MY_STRUCTURES ignores containers and roads, because these are unowned structures.
Some containers are not built because they are built based on a running count on previous containers.

**Comment by EdenPlus** [comment]
Does it work with the FIND_STRUCTURES constant?

**Comment by ghsjeff** [comment]
Yes. That argument gets ALL structures in the current room, including roads, containers, the Source Keeper Lair, etc.
